install_config/install/advanced_install.adoc

```diff
@@ -92,6 +92,15 @@ choose an example that matches your requirements, modify it to match your own
 environment, and use it as your inventory file when
 xref:running-the-advanced-installation[running the advanced installation].
 
+[[discrete]]
+[[advanced-install-image-version-policy]]
+=== Image Version Policy
+
+Images require a version number policy in order to maintain updates. See
+the
+xref:../architecture/core_concepts/containers_and_images.adoc#architecture-images-tag-policy][Image
+Version Tag Policy] section in the Architecture Guide for more information.
```

Comment:
@mburke5678 As in, putting the "Architecture Guide" in italics or something?

"see the _Architecture Guide_ for more information"